eslint-config-msal/package.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "eslint-config-msal",
+    "version": "0.0.0",
```

Comment:
0.0.0?

Reply:
We can give this any arbitrary number we want as it won't be published and won't ever be incremented. It's only defined because it's a required field.